docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,124 @@ sidebar_order: 30
-
-Broken subtraces may be caused by:
-
-- SDK sampling. Setting a sample rate that's too low may prevent the SDK from sending a transaction. We recommend [sending us all of your transaction data](/product/performance/retention-priorities/#deciding-on-your-sdk-sample-rate).
-- [Ad blockers](/platforms/javascript/troubleshooting/#dealing-with-ad-blockers) may prevent transactions in browsers being sent, but HTTP requests to backend projects will still create child transactions
-- [Rate-limiting](/product/accounts/quotas/#limiting-events) on a project may cause only some events to be sent to Sentry
-- [Project permissions](/product/accounts/membership/#restricting-access) may mean you do not have access to transactions in another project
-- [Differences in quota limits](/product/accounts/quotas/) between transactions and errors. When a quota limit is reached - for example, for transactions - an error is received, but corresponding transaction is not.
-- Exceeding the span limit. Transactions are associated via the child spans of the parent transaction, but if the number of spans exceed the limit, the association cannot be made
-
-### Multiple Roots
-
-Each trace ID should have only one root, a transaction without any parents. Automatic instrumentation should prevent multiple roots; however, if the trace ID of your transactions is being set using custom instrumentation, you may encounter multiple roots.
+The trace view is designed to be your one-stop-shop when debugging performance or errors. It gives you full context on what was happening when an error or performance issue occurred, all in one place. The waterfall trace view allows you to see everything that may have occurred during a trace, including errors, performance issues, and any profiles that may have been collected.
+
+In addition, the trace view allows you to drill down into the details of a single trace so you can visualize the high level transactions and spans that took place within that trace. This makes debugging slow services, identifying related errors, and rooting out bottlenecks easer and faster without having to navigate around multiple pages of the Sentry product.
+
+## Product Walkthrough: Trace View Page
+
+Sentry's default visualization of a trace is a waterfall-like structure, where the entries in the list are organized by chronological order as well as ancestry. This allows you to follow the order of operations as well as track sub-operations.
+
+On the left side is a list of operations, and on the right is their duration and any events, such as errors, which may have occurred in that timeframe.
+
+By default, the waterfall view shows a list of transactions. If you need more granular data and want to see spans as well, click the + button on the left and the waterfall will update and show both. The waterfall view helps surface any performance issues or profiles collected during a given timeframe.
```

Comment:
```suggestion
By default, the waterfall view shows a list of transactions. If you need more granular data and want to see spans as well, click "+" to the left of a transaction to expand the waterfall. The waterfall view helps surface any performance issues or profiles collected during a given timeframe.
```
I'm assuming from the arcade that the + only expands to show spans for that transaction, not all of them in the trace right?